docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -904,107 +904,98 @@ The indexing is rooted at a single group called indices which is a subgroup of a
 
-The config file is a .json file that defines the relative location of each part of the network:
-
-{
+### <a name="network_config">Tying it all together - the network/circuit config file
 
-    "target_simulator":"NEURON",
+The config file is a JSON file that defines the relative location of each part of the network.
 
-target_simulator can be "NEURON", “PyNN”, “NEST”, etc.  It specifies the intended target simulator of the circuit description.  For now, this field is intended as a declaration, and an implementation may decide to throw an error for unsupported targets.  In practice, mechanism and parameter names are tailored to the given target simulator.  For model_type=biophysical NEURON is supported, but not PyNN or NEST.
+It includes the following sections / values:
 
-The "manifest" section of the config file provides a convenient handle on setting variables that point to base paths.  These variables can be then used in the rest of the config file to point to various directories that share the first portion of the path.
+#### target_simulator
 
-    "manifest": {
+Can be "NEURON", "PyNN", "NEST", etc. It specifies the intended target simulator of the circuit description. For now, this field is intended as a declaration, and an implementation may decide to throw an error for unsupported targets. In practice, mechanism and parameter names are tailored to the given target simulator. For `model_type`=`biophysical` NEURON is supported, but not PyNN or NEST.
 
-        "$BASE_DIR": "/path/to/my/workspace",
-
-        "$NETWORK_DIR": "$BASE_DIR/networks",
-
-        "$COMPONENT_DIR": "$BASE_DIR/components"
-
-    },
+Example:
+```
+"target_simulator": "NEURON"
+```
 
-    "components_dir": {
+#### manifest
 
-The directory in which to find the neuronal morphology files:
+Provides a convenient handle on setting variables that point to base paths. These variables can be then used in the rest of the config file to point to various directories that share the first portion of the path. One can also use '.' in the beginning of any path to declare a path relevant to the network config file location.
 
-        "morphologies_dir": "$COMPONENT_DIR/morphologies",
+Example:
+```
+"manifest": {
+  "$BASE_DIR": "./",
+  "$NETWORK_DIR": "$BASE_DIR/networks",
+  "$COMPONENT_DIR": "$BASE_DIR/components"
+}
+```
 
-The directory in which to find the edge_types and point neuron node_types dynamics_params .json files, respectively:
+#### components_dir
 
-        "synaptic_models_dir": "$COMPONENT_DIR/synapse_dynamics",
+Base directories for different components, in particular:
 
-        "point_neuron_models_dir": "$COMPONENT_DIR/point_neuron_dynamics",
+| key | content |
+|-----|---------|
+| `morphologies_dir` | Neuronal morphology SWC files |
+| `synaptic_models_dir` | Edge types `dynamics_params` JSON files |
+| `point_neuron_models_dir` | Point neuron node_types `dynamics_params` JSON files |
```

Comment:
Path to... and the same for the rest

Reply:
I used `content` as the column header (i.e. what's inside the folder) to avoid repeating "path to" every time, but if you find it more confusing this way, I'll bring "path to" back.

Reply:
Not, it's OK I just missed it due to the plain formating and because I didn't read the text sequentially.